lib/credentials.js

```diff
@@ -32,6 +32,8 @@ var FIELDS = [
   'derivationStrategy',
   'account',
   'addressType',
+  'historyCache',
```

Comment:
Even though this works, I don't think we should add these fields to credentials. This should be handled exclusively at the export/import methods.